Apps/phgitlab/gitlab.json

```diff
@@ -0,0 +1,439 @@
+            "value_list": [],
+            "default": "",
+            "order": 0
+        },
+        "personal_access_token": {
+            "description": "Access token for a GitLab user",
+            "data_type": "password",
+            "required": true,
+            "order": 1
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate the asset configuration for connectivity using supplied configuration",
+            "verbose": "",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "list users",
+            "identifier": "list_users",
+            "description": "List all the users in the GitLab instance",
+            "verbose": "List all the users in the GitLab instance",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {},
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "list projects",
+            "identifier": "list_projects",
+            "description": "List all the projects in the GitLab instance",
+            "verbose": "List all the projects in the GitLab instance",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {},
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.*.name",
```

Comment:
Please try to avoid `*.*.` in data paths. This can be fixed in the connector file with a loop for the result data. Add each result individually to avoid the nesting

Reply:
I fixed the nesting issue in [this commit](https://github.com/phantomcyber/phantom-apps/pull/156/commits/56026540e460fab936a127d24874528fd56c91b6).